dist/constant/message.js

```diff
@@ -27,6 +27,6 @@ exports.MESSAGES = {
     USE_WEBPACK_PROMPT: 'Do you want to use webpack?',
     WELCOME_MESSAGE: 'Welcome to kintone-cli!',
     WEBPACK_ENTRY_PROMPT: 'What is the entry for webpack?',
-    USE_PROXY_PROMT: 'Do you use proxy?'
+    USE_PROXY_PROMPT: 'Do you use proxy?'
```

Comment:
It seems it's a duplication of line #23.
Please delete if it's ok.

Reply:
This problem is resolved in here: https://github.com/kintone-labs/kintone-cli/pull/118#discussion_r1273285218

Reply:
Got it.